integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/udf/TestUdf.scala

```diff
@@ -0,0 +1,103 @@
+package org.apache.carbondata.spark.testsuite.udf
+
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.test.util.QueryTest
+import org.apache.spark.sql.types.{DataType, StringType, StructField, StructType}
+import org.scalatest.BeforeAndAfterAll
+
+class TestUdf extends QueryTest with BeforeAndAfterAll {
```

Comment:
Can you rename to mention it is for complex type UDF test?

Reply:
@jackylk rename done